### ISSUE TITLE
Feature request - live preview

### ISSUE THREAD

**Issue by iGitScor** [codeAdrian/modern-fluid-typography-editor#1]
Hi Adrian,

Thank you for this exciting tool.
It would be awesome to have a live preview when "playing" with the values, what do you think?

Thanks

@iGitScor

**Comment by codeAdrian** [comment]
Hi @iGitScor That is an awesome suggestion. I'll have to think about it and come back to see how. I can approach it best.

Thanks!

**Comment by juneate** [comment]
@iGitScor Agree. I whipped up something quick in pr- #4 . I'm not a native svelte dev, so the solution might be a bit clunky, but check it out.
